### PR TITLE
Add test template for ocr-numbers

### LINKED ISSUE #1963: ocr-numbers: add test template
## Task
Create a test template file for `ocr-numbers`

### Details
1. If this is your first time contributing to this repository, welcome!
    - Contributing to a repository on GitHub requires a [fork](https://help.github.com/en/articles/fork-a-repo)
1. Create a branch on your local repository called `ocr-numbers-add-test-template`
    - Run: `git checkout -b ocr-numbers-add-test-template`
1. Clone problem-specifications repository; this is used to populate the template
    - Run: `git clone https://github.com/exercism/problem-specifications ../problem-specifications`
1. Create `exercises/ocr-numbers/.meta/template.j2`
    - See [Generator Documentation] for more information
1. Generate test file from new template
    - Run: `bin/generate_tests.py ocr-numbers`

At this point, if the generator fails, or you are otherwise stuck,
please feel free to reach out! It is perfectly acceptable to submit
an incomplete PR in order to get feedback or assistance.

### Resources
- [Contributing Guide]
- [Generator Documentation]
- [Closing issues using keywords]

[Contributing Guide]: https://github.com/exercism/python/blob/master/docs/CONTRIBUTING.md
[Generator Documentation]: https://github.com/exercism/python/blob/master/docs/GENERATOR.md
[Closing issues using keywords]: https://help.github.com/en/articles/closing-issues-using-keywords

### PR DESCRIPTION
**Pull request by tqa236**
Resolve #1963 . #2051 seems not to be active so I finished it for them.

### CODE REVIEW

File: exercises/ocr-numbers/ocr_numbers_test.py [hunk 1]
```diff
@@ -1,146 +1,122 @@
-"""Tests for the ocr-numbers exercise
-
-Implementation note:
-ocr.convert should validate its input and
-raise ValueErrors with meaningful error messages
-if necessary.
```
Comment (cmccandless) [inline exercises/ocr-numbers/ocr_numbers_test.py]: This content should go in a new `.meta/hints.md`

Comment (cmccandless) [reply exercises/ocr-numbers/ocr_numbers_test.py]: The exercise readme will need to be [regenerated](https://github.com/exercism/python/blob/master/CONTRIBUTING.md#generating-exercise-readmes) after hints.md is created/modified.

Comment (tqa236) [reply exercises/ocr-numbers/ocr_numbers_test.py]: I did that, can you check?

File: exercises/ocr-numbers/.meta/hints.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+## Hints
+
+ocr.convert should validate its input and raise ValueErrors with meaningful error messages if necessary.
```
Comment (yawpitch) [inline exercises/ocr-numbers/.meta/hints.md]: We should probably chance that to something like:

> "Your converter should validate its input and raise a ValueError with a meaningful message if it receives a string that isn't a valid OCR number."

If we mention the "convert" name we're sort of assuming the property name will never change in the canonical data.

Of course if you change this you'll need to rebuild the README.md again as well.

Comment (tqa236) [reply exercises/ocr-numbers/.meta/hints.md]: I changed that.

File: exercises/ocr-numbers/README.md [hunk 3]
```diff
@@ -62,22 +62,26 @@ Is converted to "1234567890"
...
   ||_| _|
-
+         
 ```
 
 Is converted to "123,456,789"
 
+## Hints
+
+ocr.convert should validate its input and raise ValueErrors with meaningful error messages if necessary.
```
Comment (cmccandless) [inline exercises/ocr-numbers/README.md]: You modified hints.md, so please regenerate README.md

**Pull request merged**